### PR TITLE
Written a new paragraph for var naming recommendation

### PR DESCRIPTION
**Pull request by michaelbrockus**
Written a new paragraph that would be better for this part of the document.

### CODE REVIEW

File: docs/markdown/Style-guide.md [hunk 1]
```diff
@@ -13,9 +13,14 @@ Always spaces.
...
-like to refer to your executable so something like `my_exe` would work or
-just `exe`.
+Snake case (stylized as `snake_case`) refers to the style of writing in which
+each space is replaced by an underscore (`_`) character, and the first letter of
+each word written in lowercase. It is the most common naming convention used
+in Meson build scripts as identifiers for variable.
+
+Let say you would like to refer to your executable so something like `my_exe`.
+
+Developers tend to recognize snake case values more quickly than camel case.
```
Comment (nirbheek) [inline docs/markdown/Style-guide.md:23]: FWIW I think this sentence is hard to justify and also unnecessary. A style guide doesn't have to be anything but subjective guidance.

**Comment by michaelbrockus** [comment]
So. Remove the last bit?

**Comment by nirbheek** [comment]
> So. Remove the last bit?

I would say yes.

**Comment by michaelbrockus** [comment]
Removed unneeded paragraph chunk

**Pull request merged**